Description: fix mkspecs install dir
Author: Jonathan Riddell
Origin: me
Bug: https://bugs.kde.org/show_bug.cgi?id=341289
Last-Update: 2014-11-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: qca-qt5-2.1.0.3/CMakeLists.txt
===================================================================
--- qca-qt5-2.1.0.3.orig/CMakeLists.txt
+++ qca-qt5-2.1.0.3/CMakeLists.txt
@@ -164,7 +164,7 @@ if( QCA_INSTALL_IN_QT_PREFIX )
   set(QCA_PLUGINS_INSTALL_DIR "${QT_PLUGINS_DIR}" CACHE PATH "Directory where qca plugins will install")
   set(QCA_BINARY_INSTALL_DIR "${QT_BINARY_DIR}" CACHE PATH "Directory where qca plugins will install")
   set(QCA_LIBRARY_INSTALL_DIR "${QT_LIBRARY_DIR}" CACHE PATH "Directory where qca library will install")
-  set(QCA_FEATURE_INSTALL_DIR "${QT_MKSPECS_DIR}/features" CACHE PATH "Directory where qca feature file will install")
+  set(QCA_FEATURE_INSTALL_DIR "${LIB_INSTALL_DIR}/mkspecs/features" CACHE PATH "Directory where qca feature file will install")
   if(NOT QCA_SUFFIX)
     set(QCA_INCLUDE_INSTALL_DIR "${QT_HEADERS_DIR}" CACHE PATH "Directory where qca public headers will install")
     set(QCA_PRIVATE_INCLUDE_INSTALL_DIR "${QT_HEADERS_DIR}" CACHE PATH "Directory where qca headers will install")
@@ -184,7 +184,7 @@ else( QCA_INSTALL_IN_QT_PREFIX )
   set(QCA_PLUGINS_INSTALL_DIR "${LIB_INSTALL_DIR}/${QCA_LIB_NAME}" CACHE PATH "Directory where qca plugins will install")
   set(QCA_BINARY_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/bin" CACHE PATH "Directory where qca plugins will install")
   set(QCA_LIBRARY_INSTALL_DIR "${LIB_INSTALL_DIR}" CACHE PATH "Directory where qca library will install")
-  set(QCA_FEATURE_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/mkspecs/features" CACHE PATH "Directory where qca feature file will install")
+  set(QCA_FEATURE_INSTALL_DIR "${LIB_INSTALL_DIR}/mkspecs/features" CACHE PATH "Directory where qca feature file will install")
   if(NOT QCA_SUFFIX)
     set(QCA_INCLUDE_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/include" CACHE PATH "Directory where qca public headers will install")
     set(QCA_PRIVATE_INCLUDE_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/include" CACHE PATH "Directory where qca headers will install")
